tools/audit/repo-mapper/openshift/web.dc.yaml

```diff
@@ -0,0 +1,213 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: repo-mapper
+labels:
+  template: repo-mapper
+objects:
+- apiVersion: v1
+  data:
+    Caddyfile: |
+      :2015
+
+      root * /var/www/html
+
+      file_server
+
+      encode gzip
+      log stdout
+      log stderr
+  kind: ConfigMap
+  metadata:
+    creationTimestamp: null
+    name: caddy-${NAME}-static${SUFFIX}
+    labels:
+      app: ${NAME}-static${SUFFIX}
+      deploymentconfig: ${NAME}-static${SUFFIX}
+      app-name: ${NAME}-static${SUFFIX}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}-static
+    labels: 
+      app: ${NAME}-static${SUFFIX}
+  spec:
+    lookupPolicy:
+      local: false
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}-static${SUFFIX}
+    labels: 
+      app: ${NAME}-static${SUFFIX}
+  spec:
+    minReadySeconds: 20 # should be ready for atleast 20 seconds before the container is considered available. This will allow us
+    # to catch any errors on deploy before they are available to the web
+    replicas: 1
+    selector:
+      deploymentconfig: ${NAME}-static${SUFFIX}
+    strategy:
+      resources:
+        requests:
+          cpu: '150m'
+          memory: '50Mi'
+        limits:
+          cpu: '250m'
+          memory: '105Mi'
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          deploymentconfig: ${NAME}-static${SUFFIX}
+          app: ${NAME}-static${SUFFIX}
+      spec:
+        containers:
+        - command:
+            - /bin/sh
+            - '-c'
+            - ./entrypoint.sh
+          env:
+            - name: OUTPUT_PATH
+              value: /var/www/html/reports
+            - name: SERVICE_NAME
+              value: metadata-ansible
+            - name: SERVICE_NAMESPACE
+              value: devhub-tools
+            - name: WAIT_TIME
+              value: '60'
+          image: 'docker-registry.default.svc:5000/devhub-tools/metadata-ansible:v1.0'
+          imagePullPolicy: IfNotPresent
+          name: metadata-ansible
+          ports:
+            - containerPort: 9000
+              protocol: TCP
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: /var/www/html/reports
+              name: output-dir
+        - image: ${NAME}-static:${VERSION}
+          name: repo-mapper
```

Comment:
parameterize the template, I've got the env var and name setup here: https://github.com/BCDevOps/platform-services/blob/master/tools/audit/repo-mapper/.openshift/deployment.yml

Reply:
okay we are not using this dc. could be removed. `.openshift/web.deployment.yml` is used now.